modules/generator/builder.tmpl

```diff
@@ -0,0 +1,19 @@
+package modules
+
+import (
+    {{- range .}}
+    {{.Vendor}}{{.Module | Title}} "github.com/prebid/prebid-server/modules/{{.Vendor}}/{{.Module}}"
+    {{- end}}
+)
```

Comment:
Please wrap the import statement around an if block to allow for the case of no modules:

```
{{if .}}
import (
    {{- range .}}
    {{.Vendor}}{{.Module | Title}} "github.com/prebid/prebid-server/modules/{{.Vendor}}/{{.Module}}"
    {{- end}}
)
{{end}}
```